src/lang.c

```diff
@@ -7,11 +7,17 @@
 
 #define yy_create_buffer _w32__create_buffer
 #define yy_delete_buffer _w32__delete_buffer
-#define yy_flex_debug _w32__flex_debug
+#define yy_scan_buffer _w32__scan_buffer
```

Comment:
`lang.c` is generated from `lang.l`. So no point patching that.
You could instead fix Wmake to regenerate `lang.c` (using flex) when it should. 

Now when I do `del lang.c & wmake -f watcom_w.mak`, it stupidly complains:
```
language.c(50): Error! E1055: Unable to open 'lang.c'
```

Reply:
OK, I will fix it. I supposed it is maintenance task because lang.c is in repository.

Reply:
Do you know where can I download latest flex for Windows?

Reply:
![funny-google-search-animated-gif-1](https://user-images.githubusercontent.com/945271/78457662-2d055c80-76ac-11ea-8987-c4d584fb5264.gif)